enforcer-rules/src/main/java/com/google/cloud/tools/dependencies/enforcer/LinkageCheckerRule.java

```diff
@@ -155,10 +155,11 @@ public void execute(@Nonnull EnforcerRuleHelper helper) throws EnforcerRuleExcep
             logger.warn(message);
           } else {
             logger.error(message);
+            throw new EnforcerRuleException(
+                "Failed while checking class path. See above error report.");
           }
-          throw new EnforcerRuleException(
-              "Failed while checking class path. See above error report.");
         } else {
+          // arguably shouldn't log anything on success
```

Comment:
Maybe we should convert it to a verbose log?

Reply:
That's a good idea. I need to check what other enforcer plugins do when the rule passes so we can understand what users expect.